lib/index.ejs

```diff
@@ -42,6 +38,32 @@
                     return;
                 }
 
+                // Proxy postMessage events when the SDK preview is embedded in another page.
+                // This is for development purposes only, and is NOT recommended for production
```

Comment:
Maybe explicitly call out the part of this that isn't recommended?
```suggestion
                // This is for development purposes only. Using "*" is NOT recommended for production
```

Reply:
Good call